recipes-kernel/linux/linux-96boards-bootimg.inc

```diff
@@ -0,0 +1,26 @@
+DEPENDS += "dosfstools-native mtools-native"
+
+BOOT_IMAGE_BASE_NAME = "boot-${PKGV}-${PKGR}-${MACHINE}-${DATETIME}"
+BOOT_IMAGE_BASE_NAME[vardepsexclude] = "DATETIME"
+
+do_deploy_append_hikey() {
+    # FIXME: build fastboot.efi/grubaa64.efi instead of pre-built files
+    FASTBOOT_EFI_BUILD_NUMBER=$(wget -q --no-check-certificate -O - https://ci.linaro.org/job/96boards-reference-uefi/MX_PLATFORM=hikey,MX_TYPE=RELEASE,label=docker-jessie-amd64/lastSuccessfulBuild/buildNumber)
+    FASTBOOT_EFI_URL="http://builds.96boards.org/snapshots/reference-platform/components/uefi/${FASTBOOT_EFI_BUILD_NUMBER}/release/hikey/"
+    wget --progress=dot ${FASTBOOT_EFI_URL}/AndroidFastbootApp.efi -O ${DEPLOYDIR}/fastboot.efi
+
+    GRUB_EFI_BUILD_NUMBER=$(wget -q --no-check-certificate -O - https://ci.linaro.org/job/96boards-reference-grub-efi-arm64/lastSuccessfulBuild/buildNumber)
+    GRUB_EFI_URL="https://builds.96boards.org/snapshots/reference-platform/components/grub/${GRUB_EFI_BUILD_NUMBER}"
+    wget --progress=dot ${GRUB_EFI_URL}/grubaa64.efi -O ${DEPLOYDIR}/grubaa64.efi
```

Comment:
My understanding is that wget always returns success, even the file doesnt exists. I see the risk here that  if URL gets updated on the server side, we will still generate an image without the files on it.